test/Integration/IntegrationTestBase.cs

```diff
@@ -193,13 +193,13 @@ protected void StartFunctionHost(string functionName, bool useTestFolder = false
             {
                 // This string is printed after the function host is started up - use this to ensure that we wait long enough
                 // since sometimes the host can take a little while to fully start up
-                if (e.Data.Contains("Job host started"))
+                if (e != null && !string.IsNullOrEmpty(e.Data) && e.Data.Contains($"http://localhost:{this.Port}/api"))
                 {
                     taskCompletionSource.SetResult(true);
                 }
             };
             this.TestOutput.WriteLine($"Waiting for Azure Function host to start...");
-            taskCompletionSource.Task.Wait();
+            taskCompletionSource.Task.Wait(60000);
```

Comment:
Do we still need to wait 60 seconds after the endpoint is printed?

Reply:
That's just the timeout so we don't wait forever if it doesn't finish starting up

https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.wait?view=net-6.0#system-threading-tasks-task-wait(system-int32)

Reply:
Ah gotcha, thanks!